app/models/condition.rb

```diff
@@ -147,7 +146,6 @@ def clean_times
         self.value = nil
       end
     end
-    return true
```

Comment:
For "Remove extraneous return trues" can you explain why they're extraneous now?

Reply:
https://blog.bigbinary.com/2016/02/13/rails-5-does-not-halt-callback-chain-when-false-is-returned.html